src/utils.js

```diff
@@ -0,0 +1,68 @@
+//
+// Calendar Utils
+//
+
+/**
+ * Calculates the number of weeks in the current month.
+ *
+ * @param {number} month The month.
+ * @param {number} year The year.
+ * @returns {number} The number of weeks in the month.
+ */
+export const weeksInMonth = (month, year) => {
+  // month = # (0 indexed), year = #
+  const firstDay = new Date(year, month, 1).getDay(); // day of the week of 1st
+  const daysInMonth = new Date(year, month + 1, 0).getDate();
+  return Math.ceil((firstDay + daysInMonth) / 7);
+};
+
+/**
+ * Generates the string representation of a date.
+ *
+ * @param {number} month The integer representation of the month.
+ * @param {number} day The integer representation of the day of the month.
+ * @param {number} year The integer representation of the year.
+ * @returns {string} The date as a string.
+ */
+export const getStringDate = (month, day, year) => {
+  const mm = month < 10 ? "0" + month : month;
+  const dd = day < 10 ? "0" + day : day;
```

Comment:
This looks like a great use case for [`padStart`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)!

```suggestion
  const mm = month.toString().padStart(2, "0");
  const dd = day.toString().padStart(2, "0");
```
 I think this should work....

Reply:
Yum, thanks for identifying a new (for me) method!